email-client/src/main/java/uk/gov/hmcts/cmc/email/EmailService.java

```diff
@@ -56,5 +56,7 @@ public void logSendMessageWithAttachmentFailure(RuntimeException exception, Stri
             emailData.toString(), exception.getMessage()
         );
         logger.error(errorMessage, exception);
+
+        appInsightsService.trackEvent(NOTIFICATION_FAILURE, EMAIL_SUBJECT, emailData.getSubject());
```

Comment:
instead of subject should we say, the Notification to Defendant or Notification to claimant failed.

Reply:
This is generic service, we can't find out whether it is defendant, claimant or staff